docs/design/persistent-storage.md

```diff
@@ -0,0 +1,83 @@
+# PersistentVolume
+
+This document proposes a model for managing persistent, cluster-scoped storage for applications requiring long lived data.
+
+### tl;dr
+
+Two new API kinds:
+
+A `PersistentVolume` is created by a cluster admin and is a piece of persistent storage exposed as a volume.  It is analogous to a node.
+
+A `PersistentVolumeClaim` is a user's request for a persistent volume to use in a pod. It is analogous to a pod.  
+
+One new system component:
+
+`PersistentVolumeManager` watches for new volumes to manage in the system, analogous to the node controller.  The volume manager also watches for claims by users and binds them to available volumes.
```

Comment:
Needs clarity - is this a cluster singleton or are there potentially many managers, e.g one for each flavor of device?